<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalisisVar</class>
 <widget class="QWidget" name="AnalisisVar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>911</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>911</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Análisis de Variación</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>550</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eje Horizontal:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxEjeX">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Índice</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tiempo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cantidad de Infecciones</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Porcentaje de Vacunación</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxMarcaVert">
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Marca</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cantidad de Infecciones</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Porcentaje de Vacunación</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_ejeX">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arimo</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Índice</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Graph" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>911</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>912</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arimo</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>190</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Variabilidad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cantidad de Infectados</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cantidad de Vacunados</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>911</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Graph</class>
   <extends>QOpenGLWidget</extends>
   <header>graph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
